.htaccess

```diff
@@ -27,10 +27,36 @@ Options -Indexes
   # Some servers require the RewriteBase to be set. If so, set to the correct folder.
   # RewriteBase /
 
-  RewriteCond %{REQUEST_FILENAME} !-f
+  RewriteCond %{REQUEST_URI}::$1 ^(/.+)/(.*)::\2$
+  RewriteRule ^(.*) - [E=BASE:%1]
+
+  # Sets the HTTP_AUTHORIZATION header removed by Apache
+  RewriteCond %{HTTP:Authorization} .
+  RewriteRule ^ - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]
+
+  # Redirect to URI without front controller to prevent duplicate content
+  # (with and without `/index.php`). Only do this redirect on the initial
+  # rewrite by Apache and not on subsequent cycles. Otherwise we would get an
+  # endless redirect loop (request -> rewrite to front controller ->
+  # redirect -> request -> ...).
+  # So in case you get a "too many redirects" error or you always get redirected
+  # to the start page because your Apache does not expose the REDIRECT_STATUS
+  # environment variable, you have 2 choices:
+  # - disable this feature by commenting the following 2 lines or
+  # - use Apache >= 2.3.9 and replace all L flags by END flags and remove the
+  #   following RewriteCond (best solution)
+  RewriteCond %{ENV:REDIRECT_STATUS} ^$
+  RewriteRule ^index\.php(?:/(.*)|$) %{ENV:BASE}/$1 [R=301,L]
+
+  # If the requested filename exists, simply serve it.
+  # We only want to let Apache serve files and not directories.
```

Comment:
After thinking about this I'm not sure this is what you all want.  

Originally we had  

```  
  RewriteCond %{REQUEST_FILENAME} !-f
  RewriteCond %{REQUEST_FILENAME} !-d  
```  

rewrite request filename that is not a directory and not a file. This rewrite serves files and NOT directories

The comments from symfony say  

> We only want to let Apache serve files and not directories.  

thats contradictory to what was there before no? maybe I've talked myself into a circle I dunno.